src/style.css

```diff
@@ -1 +1,121 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+.page {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.search-bar {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
```

Comment:
It's a good practice to be consistent with your margins. Here you're using `margin` to add space at the top of the `.search-bar`. Make sure you're not adding bottom margins elsewhere in your code to avoid potential margin collapse.